examples/generative/corrdiff/train.py

```diff
@@ -60,14 +55,13 @@ def main(cfg: DictConfig) -> None:
     regression_checkpoint_path = getattr(cfg, "regression_checkpoint_path", None)
     task = getattr(cfg, "task")
     outdir = getattr(cfg, "outdir", "./output")
-    data = getattr(cfg, "data", "./data")
     arch = getattr(cfg, "arch", "ddpmpp-cwb-v0-regression")
     precond = getattr(cfg, "precond", "unetregression")
 
     # parse hyperparameters
     duration = getattr(cfg, "duration", 200)
-    batch = getattr(cfg, "batch", 256)
-    batch_gpu = getattr(cfg, "batch_gpu", 2)
+    batch_size_global = getattr(cfg, "batch", 256)
+    batch_size_local = getattr(cfg, "batch_gpu", 2)
```

Comment:
How about `batch_size_gpu` instead of `batch_size_local`? I found `batch` especially very cryptic, I remember having to backtrack code quite far to figure out what it meant.

Reply:
Agree, I like `batch_size_gpu`! Feel free to resolve once updated.

Reply:
Renamed in [450a1ef](https://github.com/NVIDIA/modulus/pull/376/commits/450a1ef95c20b1feae084022a416118d980fb232). My remaining question would be if we should rename these options in the config files too. It would be consistent with the source but break compatibility with old config files.

Reply:
After discussion, changed these also in the configs in [d74f36a](https://github.com/NVIDIA/modulus/pull/376/commits/d74f36a885581efdff3aca915a32ed22cbbdeb20).